hi-ml/src/health_ml/utils/reports.py

```diff
@@ -133,6 +133,21 @@ def add_to_template(self, template_addition: str) -> None:
         self.template += template_addition
         self._add_html_end()
 
+    def add_heading(self, text: str, level: int = 2, tag_class: str = '') -> None:
```

Comment:
Can we add a test for this method in `test_reports.py`, to check that the template gets updated as expected under different values of the input params?